CONTRIBUTING.md

```diff
@@ -58,637 +123,76 @@ You can install the above tools as follows.
-| `logging` | Logging change                                    |
-
-To see the subsystem values chosen for existing commits:
-
-```sh
-$ git log --no-merges --pretty="%s" | cut -d: -f1 | sort -u
-```
-
-### Best practices for patches
-
-We recommend that each patch fixes one thing. Smaller patches are easier to
-review, more likely to be accepted and merged, and more conducive for
-identifying problems during review.
-
-A PR can contain multiple patches. These patches should generally be related
-to the [main patch](#main-patch) and the overall goal of the PR. However, it
-is also acceptable to include additional or
-[supplementary patches](#supplementary-patch) for things such as:
-
-- Formatting (or whitespace) fixes
-- Comment improvements
-- Tidy up work
-- Refactoring to simplify the codebase
-
-### Verification
-
-Correct formatting of the PR patches is verified using the
-[`checkcommits`](https://github.com/kata-containers/tests/tree/main/cmd/checkcommits)
-tool.
-
-### Examples
+word.
 
-#### Main patch
+If the commit touches many different subsystems, you might want to
+split it into multiple commits.
+In general more smaller commits are preferred to fewer larger ones.
 
-The following is an example of a full patch description for the main change that shows the required "`Fixes #XXX`" comment, which references the GitHub issue this patch resolves:
+If a commit fixes an existing GitHub issue, include `Fixes: #xxx` in the commit message.
+This will help GitHub link the commit to the issue, which shows other developers that
+work is in progress.
 
-```
-pod: Remove token from Cmd structure
-
-The token and pid data will be hold by the new Process structure and
-they are related to a container.
+#### Pull Request Format
 
-Fixes: #123
+The pull request itself must also have a name and a description.
+Like a commit, the pull request name should start with a subsystem.
+Unlike a commit, the pull request description does not become part of
+the Git log.
+The PR description can be more verbose. You might include a checklist of development steps
+or questions that you would like reviewers to consider.
+You can tag particular users or link a PR to an issue or another PR.
+If your PR depends on other PRs, you should specify this in the description.
+In general, the more context you provide in the description, the easier it will be for reviewers.
 
-Signed-off-by: Sebastien Boeuf <sebastien.boeuf@intel.com>
-```
 
-#### Supplementary patch
+### Reviews
 
-If a PR contains multiple patches, [only one of those patches](#main-patch) needs to specify the "`Fixes #XXX`" comment. Supplementary patches have an identical format to the main patch, but do not need to specify a "`Fixes #XXX`"
-comment.
+In most subprojects your PR must be approved by two reviewers before it is merged.
+Reviews are also welcome from non-reviewers.
+For information about community structure, including how to become a reviewer,
```

Comment:
The governance doc uses `maintainer` rather than `reviewer`. Consider adding a reviewer term in the governance or using maintainer here.